bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -259,6 +265,10 @@ match {{ rs_value }} {
 }
 {%- elif type.kind == "none" -%}
 {
+    // TODO: good luck to remove this
```

Comment:
I don't think we want to fix this, clippy "Why is this bad?" says about it:

> A unit value cannot usefully be used anywhere. So binding one is kind of pointless.

In our case, the let binding totally has a point !

So I'd rather remove entirely your todo comment (I don't think an explanation comment is needed here, but feel free to add one if you want ^^)